### PR TITLE
MobileFuse Adapter: add support for multiple media types

### PR DESCRIPTION
**Pull request by dtbarne**
This PR is for an overdue change since MobileFuse has supported multiple media types in a given request for quite some time now.

Thanks!

### CODE REVIEW

File: adapters/mobilefuse/mobilefuse.go [hunk 1]
```diff
@@ -187,9 +194,14 @@ func (adapter *MobileFuseAdapter) getValidImps(bidRequest *openrtb2.BidRequest,
...
-	if imps[0].Video != nil {
-		return openrtb_ext.BidTypeVideo
+func (adapter *MobileFuseAdapter) getBidType(bid openrtb2.Bid) openrtb_ext.BidType {
+	if bid.Ext != nil {
+		var bidExt BidExt
+		err := json.Unmarshal(bid.Ext, &bidExt)
+
+		if err == nil && bidExt.Mf.MediaType == "video" {
+			return openrtb_ext.BidTypeVideo
+		}
```
Comment (SyntaxNode) [inline adapters/mobilefuse/mobilefuse.go:204]: Nice. Curious, do have plans to support OpenRTB 2.6's `mtype` field?

Comment (dtbarne) [reply adapters/mobilefuse/mobilefuse.go:204]: We certainly plan to eventually but haven't seen much in the way of support from our supply or demand partners yet so it's not currently a priority.

**Review by guscarreon — COMMENTED** [review]
Looks pretty good. Left a `getBidType` question just to double-check but I'm pretty much ready to approve.

File: adapters/mobilefuse/mobilefuse.go [hunk 2]
```diff
@@ -187,9 +194,14 @@ func (adapter *MobileFuseAdapter) getValidImps(bidRequest *openrtb2.BidRequest,
...
-func (adapter *MobileFuseAdapter) getBidType(imp_id string, imps []openrtb2.Imp) openrtb_ext.BidType {
-	if imps[0].Video != nil {
-		return openrtb_ext.BidTypeVideo
+func (adapter *MobileFuseAdapter) getBidType(bid openrtb2.Bid) openrtb_ext.BidType {
+	if bid.Ext != nil {
+		var bidExt BidExt
+		err := json.Unmarshal(bid.Ext, &bidExt)
+
+		if err == nil && bidExt.Mf.MediaType == "video" {
+			return openrtb_ext.BidTypeVideo
```
[the remark this answers is not in the document]
Comment (dtbarne) [reply adapters/mobilefuse/mobilefuse.go:203]: The bidder returns that media type extension with 100% of bids (and has for most of 2022) so I don't think a legacy check would be necessary.

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**